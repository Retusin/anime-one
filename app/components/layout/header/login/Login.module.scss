.login {
  @apply cursor-pointer;

  &_button {
    @apply cursor-pointer flex items-center gap-3 hover:text-violet-500 transition-colors duration-200;
    font-size: 16px;
  }
}
