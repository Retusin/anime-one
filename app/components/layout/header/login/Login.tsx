import Link from 'next/link';
import { FC } from 'react';
import { BiLogIn } from 'react-icons/bi';

import styles from './Login.module.scss';

export const Login: FC = () => (
  <div className={styles.login}>
    <Link href='/login'>
      <button type='button' className={styles.login_button}>
        <span>
          <BiLogIn size={25} />
        </span>
        <span>Login</span>
      </button>
    </Link>
  </div>
);
