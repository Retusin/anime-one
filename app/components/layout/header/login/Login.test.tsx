import { Login } from '@/components/layout/header/login/Login';
import { render } from '@testing-library/react';

test('render menu', () => {
  const { container } = render(<Login />);
  expect(container).toMatchSnapshot();
});
