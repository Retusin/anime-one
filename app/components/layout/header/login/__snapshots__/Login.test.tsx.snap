// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render menu 1`] = `
<div>
  <div
    class="login"
  >
    <button
      class="login_button"
      type="button"
    >
      <span>
        <svg
          fill="currentColor"
          height="25"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 24 24"
          width="25"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="m13 16 5-4-5-4v3H4v2h9z"
          />
          <path
            d="M20 3h-9c-1.103 0-2 .897-2 2v4h2V5h9v14h-9v-4H9v4c0 1.103.897 2 2 2h9c1.103 0 2-.897 2-2V5c0-1.103-.897-2-2-2z"
          />
        </svg>
      </span>
      <span>
        Login
      </span>
    </button>
  </div>
</div>
`;
